core/client/fs/src/main/java/alluxio/client/file/cache/allocator/Allocator.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.allocator;
+
+import alluxio.client.file.cache.store.PageStoreDir;
+
+/**
+ * Local cache allocator.
+ */
+public interface Allocator {
```

Comment:
This does not actually allocate space for a page. It's more like choosing an appropriate dir for the page. So how about renaming it to `AllocationPolicy` or `DistributionPolicy`, something that is more "static"?